src/normalize/core.ts

```diff
@@ -103,13 +103,17 @@ export class CoreNormalizer extends SpecMapper<NormalizerParams, FacetedUnitSpec
             column: columnValue
           };
 
-          const child = this.map(childSpec, {...params, repeater: childRepeater});
-
-          child.name =
+          const childName =
+            (childSpec.name || '') +
```

Comment:
```suggestion
            (childSpec.name ?? '') +
```

Reply:
why is that more correct?  I don't think we really should use `??` everywhere when it doesn't matter?